### ISSUE TITLE
Ogmios client.delegationsAndRewards doesnt work for cardano-node 8.0.0 (client 5.5.0-rc.1)

### ISSUE THREAD

**Issue by GrzegorzDrozda** [CardanoSolutions/ogmios#313]
### What Git revision / release tag are you using?
`v5.6.0`
### Do you use any client SDK? If yes, which one?
`(package.json:    "@cardano-ogmios/client": "^5.5.0-rc.1")`
### Describe what the problem is?
Im trying to integrate ogmios with cardano-node 8.0.0 when following call is made from app:
```
            response = await client.delegationsAndRewards(
              jsonBody.stakeKeyHashes
            );
```
This call is timed out. 

In logs I have:
```
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:23.48188367Z","thread":"34494","message":{"Health":{"tag":"HealthTick","status":{"startTime":"2023-05-14T13:27:05.827769105Z","lastKnownTip":{"slot":92656052,"hash":"90862609539de9969a7ff740c3479c33c4c362ec24761068e3a04fc08c1617ec","blockNo":8779962},"lastTipUpdate":"2023-05-16T07:32:23.481419458Z","networkSynchronization":1.00000,"currentEra":"Babbage","metrics":{"activeConnections":0,"runtimeStats":{"cpuTime":810060810974,"currentHeapSize":720,"gcCpuTime":656213122164,"maxHeapSize":957},"sessionDurations":{"max":0,"mean":1019,"min":0},"totalConnections":1,"totalMessages":2,"totalUnrouted":0},"connectionStatus":"connected","currentEpoch":412,"slotInEpoch":35252}}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:45.385124328Z","thread":"40521","message":{"WebSocket":{"mode":"FullSerialization","tag":"WebSocketConnectionAccepted","userAgent":"User-Agent unknown"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:45.385609196Z","thread":"40526","message":{"WebSocket":{"contents":{"agency":"ClientAgency TokPropose","event":"send","tag":"ProposeVersions","versions":["NodeToClientV_10","NodeToClientV_11","NodeToClientV_12","NodeToClientV_13"]},"tag":"WebSocketClient"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:45.385718928Z","thread":"40525","message":{"WebSocket":{"contents":{"agency":"ClientAgency TokPropose","event":"send","tag":"ProposeVersions","versions":["NodeToClientV_10","NodeToClientV_11","NodeToClientV_12","NodeToClientV_13"]},"tag":"WebSocketClient"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:45.385971469Z","thread":"40525","message":{"WebSocket":{"contents":{"agency":"ServerAgency TokConfirm","event":"receive","tag":"AcceptVersion","version":"\"NodeToClientV_13\""},"tag":"WebSocketClient"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:45.386451675Z","thread":"40526","message":{"WebSocket":{"contents":{"agency":"ServerAgency TokConfirm","event":"receive","tag":"AcceptVersion","version":"\"NodeToClientV_13\""},"tag":"WebSocketClient"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:45.387608208Z","thread":"40532","message":{"WebSocket":{"contents":{"era":"Babbage","point":null,"query":{"delegationsAndRewards":["829bd1aee2c225ca355a45b21a036a61973fb8a49a4e4acd726e31ac"]},"tag":"StateQueryRequest"},"tag":"WebSocketStateQuery"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Error","timestamp":"2023-05-16T07:32:46.390070021Z","thread":"40521","message":{"WebSocket":{"exception":"MuxError MuxBearerClosed \"<socket: 18> closed when reading data, waiting on next header True\"","tag":"WebSocketUnknownException"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:32:46.390211646Z","thread":"40521","message":{"WebSocket":{"tag":"WebSocketConnectionEnded","userAgent":"User-Agent unknown"}},"version":"v5.6.0"}
ogmios_1                  | {"severity":"Info","timestamp":"2023-05-16T07:33:25.680798813Z","thread":"34494","message":{"Health":{"tag":"HealthTick","status":{"startTime":"2023-05-14T13:27:05.827769105Z","lastKnownTip":{"slot":92656114,"hash":"e0149412fd749d532173b3a69e3ffa85815809b42f24e06be7fe0f71c5a8cc9c","blockNo":8779963},"lastTipUpdate":"2023-05-16T07:33:25.680343476Z","networkSynchronization":1.00000,"currentEra":"Babbage","metrics":{"activeConnections":0,"runtimeStats":{"cpuTime":810290490379,"currentHeapSize":725,"gcCpuTime":656392450551,"maxHeapSize":1003},"sessionDurations":{"max":0,"mean":1012,"min":0},"totalConnections":2,"totalMessages":4,"totalUnrouted":0},"connectionStatus":"connected","currentEpoch":412,"slotInEpoch":35314}}},"version":"v5.6.0"}
```
With cardano-node 1.35.7 this works fine
### What should be the expected behavior?
Working like with 1.35.7
### If applicable, what are the logs from the server around the occurence of the problem?
Shown above.
Thank you for your help in advance

**Comment by KtorZ** [comment]
Apparently, the format of the object on-the-wire has changed and the node isn't honouring the versioning scheme of its own protocols.

This is unfortunately not an issue with Ogmios, but rather an issue with the node / networking layer of the node. I haven't been able to pin-point the exact faulty object yet. 

I'd be glad if you can raise the issue to input-output-hk/ouroboros-network or input-output-hk/cardano-node to let them know that they broke the client API with that last upgrade.

**Comment by cardano-apexpool** [comment]
I have exactly the same issue with ogmios v5.6.0 and node version 8.1.1-pre.

**Comment by AndrewWestberg** [comment]
Confirmed delegationsAndRewards call is still broken in 8.1.1-pre with ogmios v5.6.0

**Comment by AndrewWestberg** [comment]
> I have exactly the same issue with ogmios v5.6.0 and node version 8.1.1-pre when trying to query the existing UTxOs at an address.

I've created a separate issue for this one. 

https://github.com/CardanoSolutions/ogmios/issues/315

**Comment by SebastienGllmt** [comment]
Cardano Preview network just hardforked to protocol version 9 so everybody is forced to upgrade to cardano-node v8 or higher, but this issue is still a blocker
